test/stylelint-prettier-e2e.test.js

```diff
@@ -3,13 +3,22 @@ const {resolve} = require('path');
 const stripAnsi = require('strip-ansi');
 
 describe('E2E Tests', () => {
-  test('CSS/SCSS files', () => {
-    const result = runStylelint('*.{css,scss}');
+  test('CSS files', () => {
```

Comment:
I don't know why, but the order of the result are reversed sometimes, so I split the two type of files.

https://github.com/prettier/stylelint-prettier/runs/1954057143?check_suite_focus=true#step:7:9